darts/models/forecasting/block_rnn_model.py

```diff
@@ -243,6 +259,9 @@ def __init__(
             Sizes of hidden layers connecting the last hidden layer of the RNN module to the output, if any.
         dropout
             Fraction of neurons afected by Dropout.
+        activation
+            The name of the activation function to be applied between the layers of the fully connected network.
+            Options include "ReLU", "Sigmoid", "Tanh", or None for no activation. Default: None.
```

Comment:
I would not give example
```suggestion
            The name of a torch.nn activation function to be applied between the layers of the fully connected network. Default: None.
```